### ISSUE TITLE
Sighandler for SIGCHLD needed to avoid zombie processes.

### ISSUE THREAD

**Issue by nihilus** [troglobit/inadyn#267]
On my current system I do get zombie processes because SIGCHLD is not handled.
I implemented a signal handler for it using the following code:
```
static void signal_child(int sig){
	while (waitpid((pid_t) (-1), 0, WNOHANG) > 0) {}
}
```

**Comment by troglobit** [comment]
Good catch! :+1: Care to submit a pull request?

**Comment by troglobit** [comment]
Good catch! :+1: Care to submit a pull request?

**Comment by nihilus** [comment]
Yes, I am working on the EINTR it generates right now.

**Comment by nihilus** [comment]
Fixed in PR #268

**Comment by troglobit** [comment]
Merged.
